src/comm/LinkManager.cc

```diff
@@ -481,7 +481,10 @@ void LinkManager::_updateConfigurationList(void)
                 }
             } else {
                 // Lets create a new Serial configuration automatically
-                pSerial = new SerialConfiguration(QString("Pixhawk on %1").arg(portInfo.portName().trimmed()));
+		if (portInfo.description() == "AeroCore")
```

Comment:
@smithandrewc @dogmaphobic Can we get other boards that could come across with vendor id = 9900 but not be a Pixhawk? Wondering if there should be a description string check for Pixhawk (or other things) here on the other side of the if?